pkg/scheduler/framework/v1alpha1/framework.go

```diff
@@ -69,6 +70,10 @@ func NewFramework(r Registry, _ *runtime.Unknown) (Framework, error) {
 		// TODO: For now, we assume any plugins that implements an extension
 		// point wants to be called at that extension point. We should change this
 		// later and add these plugins based on the configuration.
+		if qsp, ok := p.(QueueSortPlugin); ok {
+			f.queueSortPlugins = append(f.queueSortPlugins, qsp)
```

Comment:
For the current implementation, when len(f.queueSortPlugins) reaches 1, we don't need to append more plugin.